blocks/extensions/plot.py

```diff
@@ -136,14 +130,103 @@ def preexec_fn():
             logger.info('Plotting server PID: {}'.format(self.sub.pid))
         else:
             self.sub = None
-        output_server(self.document, url=self.server_url)
+
+        self.session = Session(
+            root_url=self.server_url,
```

Comment:
I just checked and on my computer this line is not causing problems. I am using the last version of bokeh, that was released a few days ago.